Makefile

```diff
@@ -43,6 +58,10 @@ check: $(addsuffix /check,$(BUILD_SUBDIRS))
 # No need to recurse into most directories, as rm does that for us.
 .PHONY: clean
 clean: kernel/clean userspace/clean
+	rm -f runner/Cargo.lock
+	rm -f third_party/libtock-rs/Cargo.lock
+	rm -f third_party/rustc-demangle/Cargo.lock
+	rm -f tools/Cargo.lock
```

Comment:
`runner/`, `third_party/`, and `tools/` already have `Build.mk` files. We should add `clean` targets to those directories that invoke `cargo clean`, then make this `clean` depend on those (in the same way it depends on `userspace/clean` and `kernel/clean`).

Reply:
Good point, thanks for suggesting that. Fixed. See additional commit.